pkg/services/ngalert/models/alert_rule.go

```diff
@@ -240,6 +240,8 @@ type AlertRule struct {
 	PanelID         *int64  `xorm:"panel_id"`
 	RuleGroup       string
 	RuleGroupIndex  int `xorm:"rule_group_idx"`
+	Record          string
+	RecordFrom      string
```

Comment:
This is literally the same thing as Condition. But, the term "condition" doesn't make any sense for recording rules.

I'm also open to not including this field, and instead renaming Condition to something more generic. Naming is hard though, and i don't think the duplication hurts anything as it's logically a different thing.